<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>294</width>
    <height>134</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="cb_selector_fill_colour_name">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <item>
    <property name="text">
     <string>default</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>black</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>blue</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>brown</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>cyan</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>darkgray</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>gray</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>green</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>lightgray</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>lime</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>magenta</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>olive</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>orange</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>pink</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>purple</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>red</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>teal</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>violet</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>white</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>yellow</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="cb_selector_fill_colour_mix_name">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <item>
    <property name="text">
     <string>none</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>black</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>blue</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>brown</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>cyan</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>darkgray</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>gray</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>green</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>lightgray</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>lime</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>magenta</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>olive</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>orange</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>pink</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>purple</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>red</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>teal</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>violet</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>white</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>yellow</string>
    </property>
   </item>
  </widget>
  <widget class="QSlider" name="hslider_strength">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>160</width>
     <height>29</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>principal colour</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>mix colour</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>strength</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_strength">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>20</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>size</string>
   </property>
  </widget>
  <widget class="QSlider" name="hslider_mix_strength">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>160</width>
     <height>29</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>strength</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_mix_strength">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>80</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>size</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
